integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java

```diff
@@ -302,6 +302,35 @@ public static void makeDirectories(String namespace,
     });
   }
 
+  /**
+   * Create a directory in a pod in specified namespace.
+   * @param namespace The Kubernetes namespace that the pod is in
+   * @param pod The name of the Kubernetes pod where the command is expected to run
+   * @param container The container in the Pod where the command is to be run. If no
+   *                         container name is provided than the first container in the Pod is used.
+   * @param redirectToStdout copy process output to stdout
+   * @param directoryToCreate namespace in which the pod exists
+   */
+  public static void deleteDirectories(String namespace,
```

Comment:
The intent of the api is to create directories not delete, please rename the  method

Reply:
it's a api to delete the dir. Comments were wrong :(